spec/models/bus_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Bus, type: :model do
+  describe "bus model validations" do
+    let!(:bus) { FactoryBot.create(:bus) }
```

Comment:
please study difference between `let` and `let!`

Reply:
I've studied &  understood the difference between `let` and `let!`. Here I have user `let!` because for comparing uniqueness of  bus.name  it has to store the previous instance before invocation. So thats why I used that.